boards/sparkfun/micromod_nrf52840/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: micromod_nrf52840
```

Comment:
should not have nrf52840 in the name otherwise you have to build using `-DBOARD=micromod_nrf52840/nrf52840`. Then update filenames to remove the nrf52840 part

Reply:
The issue is that the board is named "MicroMod nRF52840 Processor" since "Micromod" is just the pin standard interface (like Arduino, feather, etc.). https://www.sparkfun.com/products/16984

Thus, using just "MicroMod" would not be appropriate since there are more Micromod board with different processors: https://www.sparkfun.com/micromod#processor_boards

So, how should I rename the board?

Reply:
That's what the `socs` field is used for, the board would just be `micromod` then you select which soc to build for, that could be `nrf52840` or `sam5u4` or any other CPU

Reply:
All rigth! I was thinking about it. Nice, I will modify it and force a new push. Thanks!